src/components/admin/UserProfileModal/components/header/ProfileModalBasicInfo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ProfileModalAvatar } from "components/admin/UserProfileModal/components/header/ProfileModalAvatar";
+import { ProfileModalBasicTextInfo } from "components/admin/UserProfileModal/components/header/ProfileModalBasicTextInfo";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+
+import { ModalTitle } from "components/molecules/Modal/ModalTitle";
+
+export interface ProfileModalBasicInfoProps {
+  user: WithId<User>;
+}
+
+export const ProfileModalBasicInfo: React.FC<ProfileModalBasicInfoProps> = ({
+  user,
+}) => {
+  return (
+    <div className="ProfileModalBasicInfo mt-3 text-left sm:mt-0">
```

Comment:
There is no `.scss` file imported, can we get rid of `ProfileModalBasicInfo` and the rest of the same classNames?

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8